spec/index.bs

```diff
@@ -1322,6 +1331,34 @@ interface GPUTexture {
 GPUTexture includes GPUObjectBase;
 </script>
 
+{{GPUTexture}} has the following internal slots:
+<dl dfn-type=attribute dfn-for="GPUTexture">
+    : <dfn>\[[textureSize]]</dfn> of type {{GPUExtent3D}}.
+    ::
+        The size of the {{GPUTexture}} in texels.
+
+    : <dfn>\[[mipLevelCount]]</dfn> of type {{GPUIntegerCoordinate}}.
+    ::
+        The total number of the mipmap levels of the {{GPUTexture}}.
+
+    : <dfn>\[[sampleCount]]</dfn> of type {{GPUSize32}}.
+    ::
+        The sample count of the {{GPUTexture}}.
```

Comment:
"Number of samples in each texel of the {{GPUTexture}}"

Reply:
Done